### ISSUE TITLE
e2e test flakes: traffic split tests

### ISSUE THREAD

**Issue by navidshaikh** [knative/client#694]
## What version of Knative Client?
HEAD

## What version of Knative Serving running on your cluster?
seen against nightly build of serving

## Expected Behavior
Traffic split tests pass.

## Actual Behavior

Flakes in traffic split e2e tests.
```
        traffic_split_test.go:417: assertion failed: error is not nil: expecting 3 targets, got 2 targets targets: ,echo12-rev-2,100,true|latest,echo12-rev-1,0,false separator: |
```
ref: [logs](https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_client/688/pull-knative-client-integration-tests/1233406364882046976#1:build-log.txt%3A1381)

**Comment by MIBc** [comment]
I can have a try.

**Comment by MIBc** [comment]
The test result result is ` ,echo12-rev-2,100,true|latest,echo12-rev-1,0,false`.
It seems that the cmd `kn service update echo12 --untag latest --tag echo12-rev-1=old,echo12-rev-2=latest --namespace kne2etests2` is useless.

**Comment by dsimansk** [comment]
Fixed by #701.

/close
